packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -17,6 +17,34 @@ import type {
 
 export type AuthorsMap = {[authorKey: string]: Author};
 
+interface Socials {
```

Comment:
Let's move this to `authorsSocials.ts`

I think we'll soon create a full `authors/*` subfolder, will be easier to manage long term